fatrace.8

```diff
@@ -72,6 +75,69 @@ prints the devices' major and minor number and the inode number. To examine
 such a process in more detail, you should consider using
 .BR strace (1).
 
+.RE
+
+In JSONL format, the line is a JSON-encoded object with the following fields:
+
+.TP
+.B timestamp
+The current time, if you specify the
+.B \-\-timestamp
+option.
+
+.TP
+.B comm
+Process name. This is read from /proc/pid/comm, and might be abbreviated for
+long process names.
+
+.TP
+.B comm_raw
+The raw bytes of the process name, in case it is not valid UTF-8.
+
+.TP
+.B pid
+Process ID
+
+.TP
+.B uid
+UID, if you specify the
+.B \-\-user
+option.
+
+.TP
+.B gid
+GID, if you specify the
+.B \-\-user
+option.
+
+.TP
+.B types
+Event type: \fBO\fRpen, \fBR\fRead, \fBW\fRrite, \fBC\fRlose. Events on
+directories are \fB+\fR (create), \fBD\fRelete, \fB<\fR (moved from),
+or \fB>\fR (moved to).
+Combinations are possible, such as \fBCW\fR for closing a written file, or <>
+for renaming a file within the same directory.
+
+Directory events can only be detected on Linux 5.1 or higher.
+
+.TP
+.B device
+device ID in
+.B [major,minor]
+form, if it can be determined.
```

Comment:
To me it feels a bit more json-y to format that as a `{"major": int, "minor": int}` map. That also keeps it extensible, like if we ever want to translate that to a `/dev/...` name or so. WDYT?